src/EventHandler/Filter/FilterSenders.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of MadelineProto.
+ * MadelineProto is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * MadelineProto is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with MadelineProto.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author    Daniil Gentili <daniil@daniil.it>
+ * @copyright 2016-2023 Daniil Gentili <daniil@daniil.it>
+ * @license   https://opensource.org/licenses/AGPL-3.0 AGPLv3
+ * @link https://docs.madelineproto.xyz MadelineProto documentation
+ */
+
+namespace danog\MadelineProto\EventHandler\Filter;
+
+use Attribute;
+use danog\MadelineProto\EventHandler;
+use danog\MadelineProto\EventHandler\AbstractMessage;
+use danog\MadelineProto\EventHandler\Query\ButtonQuery;
+use danog\MadelineProto\EventHandler\InlineQuery;
+use danog\MadelineProto\EventHandler\Update;
+
+/**
+ * Allow incoming or outgoing group messages made by a certain list of senders.
+ */
+#[Attribute(Attribute::TARGET_METHOD)]
+final class FilterSender extends Filter
```

Comment:
Wrong class name here :)

Reply:
Also it's probably a better idea to just share the code of the two classes now that I think of it

Reply:
like ```FilterSender(1, 2, 3, .....)``` ?